docs/usage.md

```diff
@@ -27,14 +27,14 @@ Navigate to demo alert and note the following:
     - Next steps
 
 
-- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs following actions.
+- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs the following actions.

```

Comment:
the following actions or the following action?

Reply:
`actions` would be better as playbook performing multiple things